# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/mate-menus/pkgconfig-libdir.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

Generate the pkg-config Libs from the library target, so it carries
-L${libdir} for non-standard prefixes.

--- mate-menus-1.28.1.orig/libmenu/meson.build	2025-12-18 08:07:39.000000000 +0100
+++ mate-menus-1.28.1/libmenu/meson.build	2026-09-25 19:42:02.856283700 +0200
@@ -42,7 +42,7 @@ pkg.generate(
   description: 'Desktop Menu Specification Implementation',
   requires_private: 'gio-unix-2.0',
   filebase: 'libmate-menu',
-  libraries: '-lmate-menu',
+  libraries: libmate_menu,
   subdirs: 'mate-menus',
   )
 
